openupgrade_scripts/scripts/website/17.0.1.0/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,370 @@
+---Models in module 'website'---
+new model website.controller.page
+# NOTHING TO DO
+
+---Fields in module 'website'---
+website      / ir.ui.view               / controller_page_ids (one2many): NEW relation: website.controller.page
+website      / website                  / social_tiktok (char)          : NEW hasdefault: default
+website      / website.controller.page  / _inherits                     : NEW _inherits: {'ir.ui.view': 'view_id'}
+website      / website.controller.page  / default_layout (selection)    : NEW selection_keys: ['grid', 'list'], hasdefault: default
+website      / website.controller.page  / is_published (boolean)        : NEW hasdefault: default
+website      / website.controller.page  / menu_ids (one2many)           : NEW relation: website.menu
+website      / website.controller.page  / name_slugified (char)         : NEW isfunction: function, stored
+website      / website.controller.page  / page_name (char)              : NEW required
+website      / website.controller.page  / page_type (selection)         : NEW selection_keys: ['listing', 'single'], hasdefault: default
+website      / website.controller.page  / record_domain (char)          : NEW
+website      / website.controller.page  / view_id (many2one)            : NEW relation: ir.ui.view, required
+website      / website.controller.page  / website_id (many2one)         : NEW relation: website, isrelated: related, stored
+website      / website.menu             / controller_page_id (many2one) : NEW relation: website.controller.page
+website      / website.page             / header_text_color (char)      : NEW
+website      / website.page             / url (char)                    : now required
```

Comment:
if required now, we should assure it is filled somehow or delete the not filled.